go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/aws/aws-sdk-go-v2/service/iam v1.19.0
 	github.com/aws/smithy-go v1.13.5
 	github.com/spf13/cobra v1.6.1
+	gopkg.in/yaml.v2 v2.2.8
```

Comment:
We now have _both_ v2 and v3 in our go.mod.  We should use only one and make sure the required dependencies are tidied up accordingly.